pkgs/applications/networking/instant-messengers/blink/pythonpath.patch

```diff
@@ -1,14 +1,44 @@
-diff -rupN a/blink/resources.py b/blink/resources.py
---- a/blink/resources.py	2015-03-17 03:24:06.000000000 -0600
-+++ b/blink/resources.py	2015-04-07 22:52:06.101096413 -0600
-@@ -60,14 +60,7 @@ class Resources(object):
+--- blink-2.0.0/blink/resources.py	2016-03-09 14:39:07.000000000 +0100
++++ blink-2.0.0/blink/resources-patched.py	2016-03-12 21:34:14.965476623 +0100
+@@ -1,7 +1,10 @@
++# Copyright (C) 2010-2013 AG Projects. See LICENSE for details.
```

Comment:
This got me curious (nothing wrong with it), so I checked LICENSE and noticed that this version of Blink has a different license than is mentioned in the Nix expression. 
http://devel.ag-projects.com/repositories/blink-qt/LICENSE
Apparently it is GPL3 + additional restrictions.
Maybe @jagajaga nows what we do in such cases?